### PR TITLE
Feature kvstore

### PR DESCRIPTION
**Pull request by patriya-piyawiroj**

### CODE REVIEW

File: kvstore/kvstore_server.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <grpc/grpc.h>
+#include <unordered_map>
+
+class KvstoreServiceImpl final :  public KeyValueStore::Service {
```
Comment (barath) [inline kvstore/kvstore_server.cpp]: Move the definitions to the .h file and then you can have the functions alone here.

**Comment by barath** [comment]
Just checking in -- do you have new commits you'd like reviewed?

**Comment by barath** [comment]
Looks like you have included several files that shouldn't be in the repo, including binary files and non-code files.  Could you remove those?

File: kvstore/kvstore_server.cpp (hunk 1 above)
Comment (patriya-piyawiroj) [reply kvstore/kvstore_server.cpp]: Seperated like this?

**Comment by patriya-piyawiroj** [comment]
Thank you for checking in. I've dedicated these past few days to trying to
set up the project, but still struggle with grpc linking, so I've been
hesitant to make other changes, but I'm trying to make changes where I can.
Will be at Tuesday office hours. Sorry for the late deadlines.

**Pull request merged**